torchaudio/models/wav2vec2/model.py

```diff
@@ -910,6 +915,11 @@ def hubert_pretrain_model(
 
             This option corresponds to ``final_dim`` from ``fairseq``.
 
+        feature_grad_mult (float):
+            Whether to scale down the gradient of convolutional feature extraction layers.
```

Comment:
the language here implies that `feature_grad_mult` is of type `bool` when it's actually of type `float`. something like "factor to scale convolutional feature extraction layer gradients by" would be clearer

Reply:
Also probably worth mentioning that forward path is not affected.